requirements-test.txt

```diff
@@ -2,8 +2,8 @@
 pip>=9  # minimum installation requirements
 setuptools>=21  # minimum installation requirements
 coverage==4.5.3 # pytest-cov
-pytest==4.4.0
-psycopg2==2.8; platform_python_implementation != "PyPy"
-psycopg2cffi==2.8.1; platform_python_implementation == "PyPy"
 port-for==0.4
+psycopg2-binary==2.8; platform_python_implementation != "PyPy"
```

Comment:
I did this because it will make the tests mildly faster and easier to install, can revert if you so wish.